bitcoin/Cargo.toml

```diff
@@ -50,7 +50,7 @@ hex_lit = "0.1.1"
 serde_json = "1.0.0"
 serde_test = "1.0.19"
 serde_derive = "1.0.103"
-secp256k1 = { version = "0.25.0", features = ["recovery"] }
+secp256k1 = { git = "https://github.com/rust-bitcoin/rust-secp256k1", tag = "2023-02-01--fix-hashes", features = ["recovery"] }
```

Comment:
I couldn't get it to work. Tried again just now and still no dice, does it matter? Here is what I used in case the problem is something obvious
```
[patch.crates-io.secp256k1]
gitt = "https://github.com/rust-bitcoin/rust-secp256k1"
tag = "2023-02-01--fix-hashes"
```

And the error
```
warning: Patch `secp256k1 v0.26.0 (https://github.com/rust-bitcoin/rust-secp256k1?tag=2023-02-01--fix-hashes#b38b94b7)` was not used in the crate graph.
Check that the patched package version and available features are compatible
with the dependency requirements. If the patch has a different version from
what is locked in the Cargo.lock file, run `cargo update` to use the new
version. This may also occur with an optional dependency that is not enabled.
```

Reply:
The reason is that releasing would be easier - we could keep the patch in and just bump the version. Although now that I think about it, maybe not.